diff --git a/stdlib.go b/stdlib.go
index 52b5e25..dd8eaa3 100644
--- a/stdlib.go
+++ b/stdlib.go
@@ -685,7 +685,7 @@ const STDLIB = "#!/usr/bin/env bash\n" +
 	"# (e.g `use nix -p ocaml`).\n" +
 	"#\n" +
 	"use_nix() {\n" +
-	"  direnv_load nix-shell --show-trace \"$@\" --run \"$(join_args \"$direnv\" dump)\"\n" +
+	"  direnv_load nix-shell \"$@\" --run \"$(join_args \"$direnv\" dump)\"\n" +
 	"  if [[ $# == 0 ]]; then\n" +
 	"    watch_file default.nix\n" +
 	"    watch_file shell.nix\n" +
diff --git a/stdlib.sh b/stdlib.sh
index 4ab1b48..bc071f5 100755
--- a/stdlib.sh
+++ b/stdlib.sh
@@ -683,7 +683,7 @@ use_node() {
 # (e.g `use nix -p ocaml`).
 #
 use_nix() {
-  direnv_load nix-shell --show-trace "$@" --run "$(join_args "$direnv" dump)"
+  direnv_load nix-shell "$@" --run "$(join_args "$direnv" dump)"
   if [[ $# == 0 ]]; then
     watch_file default.nix
     watch_file shell.nix
