diff --git a/stdlib.sh b/stdlib.sh
index 64bacc2..f6c8d1b 100755
--- a/stdlib.sh
+++ b/stdlib.sh
@@ -1298,7 +1298,7 @@ use_nix() {
     ["TEMPDIR"]=${TEMPDIR:-__UNSET__}
     ["terminfo"]=${terminfo:-__UNSET__}
   )
-  direnv_load nix-shell --show-trace "$@" --run "$(join_args "$direnv" dump)"
+  direnv_load nix-shell "$@" --run "$(join_args "$direnv" dump)"
   for key in "${!values_to_restore[@]}"; do
     local value=${values_to_restore[$key]}
     if [[ $value == __UNSET__ ]]; then
